pkg/mod/fs.go

```diff
@@ -23,6 +24,7 @@ func (fs *Fs) Open(name string) (afero.File, error) {
 		return f, nil
 	}
 
+	name = fs.root + string(os.PathSeparator) + name
```

Comment:
filepath.Join() here and below

Reply:
I was going to say for that I need to create a slice, so I don't want to do that, but that's `strings.Join()`.

Will do.

Reply:
Fixed in 40dc688